rama-cluster/download_rama.sh

```diff
@@ -7,4 +7,17 @@ cd /data/rama
 curl --connect-timeout 180 'http://${conductor_ip}:8888/d/jar/download/rama.zip' \
      --output rama.zip
 
+until curl -s -f 'http://${conductor_ip}:8888/d/jar/download/rama.zip' \
```

Comment:
Should the previous `curl` command be deleted?

Reply:
Ahh yes, it absolutely should.